eslint.config.ts

```diff
@@ -0,0 +1,54 @@
+import tinyPlugin from '@tinymce/eslint-plugin';
+import securityNode from 'eslint-plugin-security-node';
```

Comment:
This one is likely not needed. It's designed for node backend things mostly, it has one rule in it that we use for the random id it was introduced in this ticket https://ephocks.atlassian.net/browse/TINY-10946 there might be some other option we can use for that check to filter out `Math.random` calls.

Reply:
We are really using only the `detect-insecure-randomness` rule, which prevents the use of `Math.random`. I think we can just add this rule to our plugin and call it a day. The security plugins are there for server code really.

[Here](https://github.com/eslint-community/eslint-plugin-security/tree/main) is a maintained security plugin but it doesn't have a rule for Math.random

Reply:
We agreed in the checkin to remove this plugin, especially that CodeQL can detect [rule around insecure randomness](https://codeql.github.com/codeql-query-help/javascript/js-insecure-randomness/)